deploy/kubernetes/dolphinscheduler/values.yaml

```
@@ -506,10 +506,15 @@ master:
     MASTER_TASK_COMMIT_INTERVAL: "1s"
     # -- master state wheel interval, the unit is second
     MASTER_STATE_WHEEL_INTERVAL: "5s"
-    # -- Master max cpuload avg, only higher than the system cpu load average, master server can schedule
-    MASTER_MAX_CPU_LOAD_AVG: "1"
-    # -- Master reserved memory, only lower than system available memory, master server can schedule, the unit is G
-    MASTER_RESERVED_MEMORY: "0.3"
+    MASTER_SERVER_LOAD_PROTECTION_ENABLED: true
+    # Master max cpu usage, when the master's cpu usage is smaller then this value, master server can execute workflow.
```

```suggestion
    # -- Master max cpu usage, when the master's cpu usage is smaller then this value, master server can execute workflow.
```

Refer to [helm-docs](https://github.com/norwoodj/helm-docs), comments need to be added with -- for them to be recognized, then they will be automatically updated to README.md by helm-docs.

---

Done